parquet-avro/src/main/java/org/apache/parquet/avro/AvroSchemaConverter.java

```diff
@@ -300,9 +300,8 @@ private Schema convertFields(String name, List<Type> parquetFields, Map<String,
     Integer nameCount = names.merge(name, 1, (oldValue, value) -> oldValue + 1);
     for (Type parquetType : parquetFields) {
       Schema fieldSchema = convertField(parquetType, names);
-      if (parquetType.isRepetition(REPEATED)) {
-        throw new UnsupportedOperationException(
-            "REPEATED not supported outside LIST or MAP. Type: " + parquetType);
+      if (parquetType.isRepetition(REPEATED)) { // If a repeated field is ungrouped, treat as REQUIRED per spec
+        fields.add(new Schema.Field(parquetType.getName(), Schema.createArray(fieldSchema)));
```

Comment:
This constructor is not available in avro 1.8. This leads to a runtime error
```
Cause: java.lang.NoSuchMethodError: 'void org.apache.avro.Schema$Field.<init>(java.lang.String, org.apache.avro.Schema)'
```
When using `parquet-avro` 1.14 with avro version 1.8